sdk/python/dev/pyproject.toml

```diff
@@ -2,7 +2,11 @@
 name = "main"
 version = "0.0.0"
 requires-python = ">=3.12"
-dependencies = ["dagger-io"]
+dependencies = [
+    "dagger-io",
+    # Temporary, until in SDK's next release.
+    "opentelemetry-exporter-otlp-proto-http>=1.27.0",
```

Comment:
We need to keep our modules compatible with both release and dev versions for CI. 😮‍💨